deploy/docker/api.py

```
@@ -453,11 +469,55 @@ async def handle_crawl_request(
+        except Exception as bulk_error:
+            logger.warning("Bulk crawl failed, retrying per-URL: %s", bulk_error)
+            fallback_results = []
+            for url in urls:
+                try:
+                    single_result = await crawler.arun(
+                        url=url,
+                        config=crawler_config,
+                        dispatcher=dispatcher,
+                    )
+                    fallback_results.append(single_result)
+                except Exception as url_error:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)